GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/UI/VFSAboutWindowController.m

```diff
@@ -0,0 +1,55 @@
+#import "VFSAboutWindowController.h"
+
+@interface VFSAboutWindowController ()
+
+@property (strong) VFSProductInfoFetcher *productInfoFetcher;
+
+@end
+
+@implementation VFSAboutWindowController
+
+- (instancetype)initWithProductInfoFetcher:(VFSProductInfoFetcher *)productInfoFetcher
+{
+    if (productInfoFetcher == nil)
+    {
+        self = nil;
+    }
+    else if (self = [super initWithWindowNibName:@"VFSAboutWindowController"])
+    {
+        _productInfoFetcher = productInfoFetcher;
+    }
+    
+    return self;
+}
+
+- (NSString *)vfsforgitVersion
+{
+    NSString *version;
+    NSError *error;
+    if ([self.productInfoFetcher tryGetVFSForGitVersion:&version error:&error])
```

Comment:
Is this (the tryXXXX) a common pattern for objective-c? I thought if a method had an error out parameter, it was expected to have a `BOOL` return value?

Reply:
As discussed, made a note of this in the [TODO list](https://github.com/microsoft/VFSForGit/issues/1262). Will incorporate in an upcoming PR.